server/lib/ocr/klippa/KlippaOCRService.ts

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios';
+
+import { reportErrorToSentry, reportMessageToSentry } from '../../sentry';
+import { ExpenseOCRParseResult, ExpenseOCRService } from '../ExpenseOCRService';
+
+import { KlippaParseAccountingDocumentResponse } from './types';
+
+const KLIPPA_BASE_URL = 'https://custom-ocr.klippa.com/api/v1';
+
+export class KlippaOCRService implements ExpenseOCRService {
```

Comment:
Should this class be a singleton or are we OK with having this being instantiated and garbage collected when needed?

Reply:
So far no need for a singleton, it's safe to be garbage collected then re-instantiated